google-cloud-tools-plugin/build.gradle

```diff
@@ -22,7 +22,7 @@ sourceSets.test.resources.srcDirs = ['testResources']
 
 intellij {
     pluginName = 'google-cloud-tools'
-    plugins 'Groovy','gradle', 'git4idea', 'properties', 'junit'
+    plugins 'Groovy','gradle', 'git4idea', 'properties', 'junit', 'maven'
```

Comment:
just cause you already changed this file, any way you can add a space between 'Groovy','gradle'